src/components/Feed/Events/GalleryUpdatedFeedEvent.tsx

```diff
@@ -0,0 +1,192 @@
+              <Link href={galleryPagePath} passHref>
+                <StyledEventLabel>{event?.gallery?.name || 'gallery'}</StyledEventLabel>
+              </Link>
+              <StyledTime>{getTimeSince(event.eventTime)}</StyledTime>
+            </BaseM>
+          </HStack>
+        </StyledEventHeader>
+        <StyledEventContent>
+          <VStack gap={16}>
+            {caption && (
+              <StyledCaptionContainer gap={8} align="center">
+                <BaseM>{caption}</BaseM>
+              </StyledCaptionContainer>
+            )}
+            {event?.subEventDatas?.map((subEvent, index) => {
+              switch (subEvent.__typename) {
+                case 'CollectionCreatedFeedEventData':
+                  return (
+                    <ReportingErrorBoundary fallback={<></>} dontReport>
+                      <CollectionCreatedFeedEvent
+                        key={index}
+                        eventDataRef={subEvent}
+                        queryRef={query}
+                        caption={null}
+                        isSubEvent
+                      />
+                    </ReportingErrorBoundary>
+                  );
+                case 'CollectionUpdatedFeedEventData':
+                  return (
+                    <ReportingErrorBoundary fallback={<></>} dontReport>
+                      <CollectionUpdatedFeedEvent
+                        eventDataRef={subEvent}
+                        queryRef={query}
+                        isSubEvent
+                      />
+                    </ReportingErrorBoundary>
+                  );
+                case 'CollectorsNoteAddedToTokenFeedEventData':
+                  return (
+                    <ReportingErrorBoundary fallback={<></>} dontReport>
+                      <CollectorsNoteAddedToTokenFeedEvent
+                        eventDataRef={subEvent}
+                        queryRef={query}
+                        isSubEvent
+                      />
+                    </ReportingErrorBoundary>
+                  );
+                case 'TokensAddedToCollectionFeedEventData':
+                  return (
+                    <ReportingErrorBoundary fallback={<></>} dontReport>
+                      <TokensAddedToCollectionFeedEvent
+                        caption={null}
+                        eventDataRef={subEvent}
+                        queryRef={query}
+                        isSubEvent
+                      />
+                    </ReportingErrorBoundary>
+                  );
+                case 'CollectorsNoteAddedToCollectionFeedEventData':
+                  return (
+                    <ReportingErrorBoundary fallback={<></>} dontReport>
+                      <CollectorsNoteAddedToCollectionFeedEvent
+                        eventDataRef={subEvent}
+                        queryRef={query}
+                        isSubEvent
+                      />
+                    </ReportingErrorBoundary>
+                  );
```

Comment:
This one felt redudant. I'll find other way to simplify it. 

Lmk if you guys have any suggestion on this